### ISSUE TITLE
NFRs - Customer Present - Process/method to prevent misuse/abuse

### ISSUE THREAD

**Issue by bmangalaganesh** [ConsumerDataStandardsAustralia/standards#86]
Hi,

The specs define **Customer Present** as follows -  Authenticated API requests made in direct response to interactions by the end customer using the digital services of the data recipient will be considered _“Customer Present”_. 

The expected response times for _customer present_ requests are significantly shorter  than the _unattended_ requests ( **1 or 1.5 seconds vs 4 seconds**).

What is the mechanism/procedures that are kept in place to prevent misuse of customer present requests by ADRs?

In other words, the Data Holder when they receive any request with **x-fapi-customer-ip-address** header info present is expected to provider a faster response (while they do not truly know if there is a customer at the other end of the ADR/TPP app)

**Comment by CDR-API-Stream** [comment]
This comment has been transferred to the maintenance repository.  It would be appreciated if new issues or questions for clarification could be posted there.

The new issue can be found at:
https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/11
